composer.json

```diff
@@ -41,7 +41,8 @@
         "mikey179/vfsstream": "^1.6.8",
         "ocramius/proxy-manager": "^2.8",
         "phpbench/phpbench": "^0.17",
-        "phpunit/phpunit": "^7.5.20"
+        "phpunit/phpunit": "^7.5.20",
+        "vimeo/psalm": "^3.13"
```

Comment:
needs composer `v2`.

Because ...
```
$ sudo composer self-update --stable
Upgrading to version 1.10.10 (stable channel).
$ rm composer.lock & composer install
$ composer require --dev vimeo/psalm
Using version ^3.13 for vimeo/psalm
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Conclusion: don't install vimeo/psalm 3.13.1
    - Conclusion: remove ocramius/package-versions 1.9.0
    - Installation request for vimeo/psalm ^3.13 -> satisfiable by vimeo/psalm[3.13.0, 3.13.1].
    - Conclusion: don't install ocramius/package-versions 1.9.0
    - vimeo/psalm 3.13.0 requires composer/package-versions-deprecated ^1.8.0 -> satisfiable by composer/package-versions-deprecated[1.10.99, 1.10.99.1, 1.8.0, 1.8.1, 1.8.2].
    - don't install composer/package-versions-deprecated 1.10.99|don't install ocramius/package-versions 1.9.0
    - don't install composer/package-versions-deprecated 1.10.99.1|don't install ocramius/package-versions 1.9.0
    - don't install composer/package-versions-deprecated 1.8.0|don't install ocramius/package-versions 1.9.0
    - don't install composer/package-versions-deprecated 1.8.1|don't install ocramius/package-versions 1.9.0
    - don't install composer/package-versions-deprecated 1.8.2|don't install ocramius/package-versions 1.9.0
    - Installation request for ocramius/package-versions (locked at 1.9.0) -> satisfiable by ocramius/package-versions[1.9.0].


Installation failed, reverting ./composer.json to its original content.
```

Reply:
so, if we can allow to use composer v2 as dev tools, 
`.travis.yml` also needs `composer self-update --2`, OR 
needs introducing specific GitHub Action for psalm.

Reply:
You probably want to pin to `ocramius/proxy-manager:^2.8.0`, as that's compatible with both composer releases